### ISSUE TITLE
Using with Timatron's Video Looper script produces visible terminal debug text between clips

### ISSUE THREAD

**Issue by dirtpi** [mjfwalsh/omxplayer#6]
This is not so much an issue with your fork of omxplayer itself, but I am hoping you can help with this issue I am encountering while using your fork along with [Timatron's Looper Script](https://github.com/timatron/videolooper-raspbian)

I've been using the looper script with the standard omxplayer install on jessie/stretch for years and it has always been able to step between video files with a completely black screen in between when one video stops and the next one starts. 

I wanted to re-build this setup on bullseye so I can run this setup on newer pi models.  I was able to get the standard omxplayer installed and it works the same as before and goes completely black between video files without any terminal text showing.  Once I realized the OSD was no longer working I started researching and found your fork.  I set up everything with your fork of omxplayer and it has such great features, however when a video ends there is some visible terminal text before the next clip starts.  The problem seems to be made worse if the video formats differ, such as going from an .mp4 file to .avi.  

I know it's sort of a random issue, but I'd really appreciate any help you could offer in eliminating the terminal text, as I use this set up for projects and presentations where I prefer a more polished look.  

Thanks!

**Comment by dirtpi** [comment]
![IMG_1139](https://github.com/mjfwalsh/omxplayer/assets/158029717/1522be70-0f29-4ce8-b6fd-0e613da188e8)
Sorry for the bad picture but this is the terminal text I see between clips.

**Comment by mjfwalsh** [comment]
The first problem here is that the script and omxplayer are working against each other, as my fork automatically queues the next file in a directory to play when it's finished playing one.

You could always try running omxplayer with the `-a` command line option. That way it should behave more like the original. (It disables the autoplaylist feature). You could also try using `-b` which should set a black background.

**Comment by dirtpi** [comment]
I really appreciate your response.  

I always keep -b on the command line, but unfortunately it doesn’t have any effect on the terminal text in between clips.

I can try -a, but I don’t want to lose the position remembering feature you’ve implemented. 

Now that you’ve shed some light on how the playlists are working maybe I can drop the looper script.  Once all videos in the folder have been played will the playlist start over again?  

I want to be able to set a list of videos playing indefinitely, while also being able to press Q to skip to the next video if needed.

With the looper script I am also able to shuffle the videos in random order which is another feature I like having in case I have a huge folder of TV show episodes for example. 

Thank You

**Comment by dirtpi** [comment]
I've been trying a lot of different things, and the only thing I've found that works to 100% cure the issue is connecting to the pi through SSH from my pc and typing on the terminal : sudo sh -c "TERM=linux setterm -foreground black -clear all >/dev/tty0"

After doing this, when I press Q on the keyboard attached to the pi, it will go to the next video perfectly without any visible text in between.  

I've been trying to add the line into the looper script and it almost cures the problem, but you see the text flash for a quick moment before going black and starting the next video.  

Please let me know if you have any ideas.

**Comment by dirtpi** [comment]
<img width="606" alt="omxplayer text" src="https://github.com/mjfwalsh/omxplayer/assets/158029717/fb03c78e-f7cd-497f-ba29-971b95b8a1d1">
I got a clearer picture of the terminal messages that are popping up.  They seem to be caused when playing older .avi files.  Even though there are some errors they still play back just fine.  For whatever reason the standard omxplayer will hide these errors.  I believe they are coming from ffmpeg.

I've tried to add some commands such as:
ffmpeg -hide_banner -loglevel panic
ffmpeg -y -i "$in_file" "$out_file" > /dev/null 2>&1

but they aren't having any effect.  Is there anywhere in the compiling of omxplayer where ffmpeg outputs are able to be suppressed?

Sorry to keep bothering you about this.
